modules/core/src/lib/deck.js

```diff
@@ -272,6 +272,22 @@ export default class Deck {
     return redraw;
   }
 
+  redraw() {
+    // Check if we need to redraw
+    const redrawReason = this.needsRedraw({clearRedrawFlags: true});
+
+    if (!redrawReason) {
+      return;
```

Comment:
Maybe provide the option to skip the check (e.g. a `force` argument that defaults to false)? I can imagine scenarios where that would be useful.